hudi-hadoop-common/src/main/java/org/apache/hudi/common/util/HFileUtils.java

```diff
@@ -232,6 +232,19 @@ public byte[] serializeRecordsToLogBlock(HoodieStorage storage,
     return baos.toByteArray();
   }
 
+  /**
+   * Print the meta fields of the record of interest
+   */
+  private void printRecord(String msg, byte[] bs, Schema schema) throws IOException {
+    GenericRecord record = HoodieAvroUtils.bytesToAvro(bs, schema);
+    if (schema.getField(HoodieRecord.RECORD_KEY_METADATA_FIELD) != null) {
+      LOG.error(String.format("%s: Hudi meta field values -> Record key: %s, Partition Path: %s, FileName: %s, CommitTime: %s, CommitSeqNo: %s", msg,
```

Comment:
Can you use `{}` to follow the logging best practices. The `toString` can also be removed